collectors/statsd.plugin/k6.md

```diff
@@ -0,0 +1,72 @@
+<!--
+title: "K6 load test monitoring with Netdata"
+custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/stats.d.plugin/k6.md
+sidebar_label: "K6 Load Testing"
+-->
+
+# K6 Load Testing monitoring with Netdata
+
+Monitors the impact of load testing experiments performed with [K6](https://k6.io/).
+
+You can read more about the metrics that K6 sends in the [K6 documentation](https://k6.io/docs/using-k6/metrics/). 
+
+## Requirements
+
+-   When running the k6 experiment, specify a [StatsD output](https://k6.io/docs/results-visualization/statsd/).
+    -   Tip: K6 currently supports tags only with [datadog output](https://k6.io/docs/results-visualization/datadog/), which is in essence StatsD. Netdata can be used with both.
+
+## Metrics
+
+### HTTP Requests
+ 
+    Number of HTTP requests that K6 generates, per second.
```

Comment:
```suggestion
Number of HTTP requests that K6 generates, per second.
```


<img width="709" alt="Screenshot 2021-04-26 at 22 08 32" src="https://user-images.githubusercontent.com/22274335/116137226-f4338d80-a6db-11eb-9d01-98e2b96391ed.png">

Other changes after I applied markdown formatting with hard wrap at 120. If you don't use any, I suggest looking into it - a pretty handy thing.




```diff
diff --git a/k6.md b/k6.md
index f94812c..eaaf998 100644
--- a/k6.md
+++ b/k6.md
@@ -8,24 +8,26 @@ sidebar_label: "K6 Load Testing"

 Monitors the impact of load testing experiments performed with [K6](https://k6.io/).

-You can read more about the metrics that K6 sends in the [K6 documentation](https://k6.io/docs/using-k6/metrics/).
+You can read more about the metrics that K6 sends in the [K6 documentation](https://k6.io/docs/using-k6/metrics/).

 ## Requirements

--   When running the k6 experiment, specify a [StatsD output](https://k6.io/docs/results-visualization/statsd/).
-    -   Tip: K6 currently supports tags only with [datadog output](https://k6.io/docs/results-visualization/datadog/), which is in essence StatsD. Netdata can be used with both.
+- When running the k6 experiment, specify a [StatsD output](https://k6.io/docs/results-visualization/statsd/).
+    - Tip: K6 currently supports tags only with [datadog output](https://k6.io/docs/results-visualization/datadog/),
+      which is in essence StatsD. Netdata can be used with both.

 ## Metrics

 ### HTTP Requests
-
-    Number of HTTP requests that K6 generates, per second.
+
+Number of HTTP requests that K6 generates, per second.

 ### Failed HTTP Requests

 Number of failed HTTP requests that K6 generates, per second.

 ### Virtual Active Users
+
 Current number of active virtual users.

 ### Iteration Duration
@@ -41,8 +43,9 @@ The number of iterations that could not be started either due to lack of Virtual
 The amount of data received and sent.

 ### TTP Requests total duration
-
+
 The total duration it took for a round-trip of an HTTP request. It includes:
+
 - Blocked HTTP requests: time spent blocked before initiating the request
 - Connecting HTTP requests: time spent establishing TCP connection to the remote host
 - Sending HTTP requests: time spent sending data to the remote host
@@ -51,9 +54,11 @@ The total duration it took for a round-trip of an HTTP request. It includes:

 ### HTTP duration metrics

-Different metrics on the HTTP request as defined by K6. The HTTP request duration is defined by K6 as: `HTTP sending request` + `HTTP receiving request` + `HTTP waiting request`.
+Different metrics on the HTTP request as defined by K6. The HTTP request duration is defined by K6
+as: `HTTP sending request` + `HTTP receiving request` + `HTTP waiting request`.

 Metrics:
+
 - Median
 - Average
 - Max
@@ -65,8 +70,12 @@ Metrics:

 The collector is preconfigured and defined in `statsd.plugin/k6.conf`.

-Due to being a StatsD collector, you only need to define the configuration file and then send data to Netdata using the StatsD protocol.
+Due to being a StatsD collector, you only need to define the configuration file and then send data to Netdata using the
+StatsD protocol.

-If Netdata is running on the same machine as K6, no further configuration is required. Otherwise, you will have to [point K6](https://k6.io/docs/results-visualization/statsd/) to your node and make sure that the K6 process can reach Netdata.
+If Netdata is running on the same machine as K6, no further configuration is required. Otherwise, you will have
+to [point K6](https://k6.io/docs/results-visualization/statsd/) to your node and make sure that the K6 process can reach
+Netdata.

-The default namespace that is used in the configuration is `k6`. If you change it in K6, you will have to change it as well in the configuration file `k6.conf`.
+The default namespace that is used in the configuration is `k6`. If you change it in K6, you will have to change it as
+well in the configuration file `k6.conf`.

```